multidict/__init__.py

```diff
@@ -57,3 +58,15 @@
 
 
 upstr = istr
+
+# Inspired by Numpy
+
+
+def get_include() -> str:
+    """Get multidict headers for compiling multidict with other
+    C Extensions or cython code"""
```

Comment:
This doesn't seem PEP 257-compliant. Could you organize it as a summary line and a separate description?